templates/autorizacija.html

```diff
@@ -5,13 +5,13 @@ <h3 class="title">Pieteikties</h3>
         <form method="POST" action="/pieteikties">
```

Comment:
```suggestion
      {% with messages = get_flashed_messages() %}
      {% if messages %}
          <div class="notification is-danger">
              {{ messages[0] }}
          </div>
      {% endif %}
      {% endwith %}
        <form method="POST" action="/pieteikties">
```
Būtu labi pievienot bloku, kur parādās "flashed" ziņas par paroles/lietotājvārda kļūdam